src/components/QuickOpenModal.js

```diff
@@ -319,7 +315,10 @@ export class QuickOpenModal extends Component<Props, State> {
             const t = title[titleElem];
             if (q === t || queryUpper[queryElem] === t) {
               styledResults.push(
-                <span key={q + titleElem} style={styles}>
+                <span
+                  key={q + titleElem}
+                  className="result-item title matching-letter"
```

Comment:
You should just need `matching-letter` here. The rest is already there in terms of classes. The `matching-letter` is already inside a container with the class `title` which in turn is inside a container classed `result-item`.